libs/service-portal/settings/access-control/src/screens/GrantAccess/GrantAccess.tsx

```diff
@@ -43,41 +43,56 @@ const IdentityQuery = gql`
 `
 
 function GrantAccess() {
-  const { handleSubmit, control, errors, setValue, watch } = useForm()
-  const [createAuthDelegation, { loading }] = useMutation<Mutation>(
-    CreateAuthDelegationMutation,
+  const [name, setName] = useState('')
+  const { handleSubmit, control, errors, watch } = useForm({ mode: 'onChange' })
+  const [
+    createAuthDelegation,
+    { loading: mutationLoading },
+  ] = useMutation<Mutation>(CreateAuthDelegationMutation, {
+    refetchQueries: [{ query: AuthDelegationsQuery }],
+  })
+  const [getIdentity, { data, loading: queryLoading }] = useLazyQuery<Query>(
+    IdentityQuery,
     {
-      refetchQueries: [{ query: AuthDelegationsQuery }],
+      onError: (error) => {
+        toast.error(
+          formatMessage({
+            id: 'service.portal.settings.accessControl:grant-identity-error',
+            defaultMessage: 'Enginn notandi fannst með þessa kennitölu.',
```

Comment:
"Fann ekki einstakling með þessa kennitölu."?

Meikar sense að þetta sé onError? Finnst eins og að queryið ætti að skila null ef kennitalan er ekki skráð.

Getum við hent inn graphql query til að filter-a einstaklinga. Svo það gleymist ekki þegar við tengjum fyrirtækjaskrá hinum megin.

Reply:
*graphql input argument

Reply:
You're correct, this should be `null`.
But it isn't handled by the national registry xroad service; so fixing that would've needed refactoring in bunch of places. I'll create a ticket for it and we should tackle that as a separate PR.

Reply:
https://github.com/orgs/island-is/projects/9#card-67950820

Reply:
Sounds good.